docs/markdown/Python/python-goals/python-test-goal.md

```diff
@@ -196,6 +173,82 @@ python_tests(
 )
 ```
 
+Batching and parallelism
+------------------------
+
+By default, Pants will schedule concurrent `pytest` runs for each `python_test` target passed to the `test` goal. This approach provides parallelism with fine-grained caching, but can have drawbacks in some situations:
```

Comment:
For the reader, `python_test` target might scan as `python_tests` target, so they'd expect all the tests in a dir to run in a single process. Maybe just say "for each test file"? It's more evocative to the average user.